voluptuous/schema_builder.py

```diff
@@ -937,8 +954,7 @@ class Msg(object):
 
     def __init__(self, schema: Schemable, msg: str, cls: typing.Optional[typing.Type[Error]] = None) -> None:
         if cls and not issubclass(cls, er.Invalid):
-            raise er.SchemaError("Msg can only use subclases of"
-                                 " Invalid as custom class")
+            raise er.SchemaError("Msg can only use subclases of" " Invalid as custom class")
```

Comment:
I've had this before, the thing with black is that it 100% guarantees that the AST stays the exact same between transformations, and merging two strings would break that promise on python 3.7- (see https://github.com/psf/black/issues/26 )

But I think it should detect that we're not running on python 3.7- and merge them, so I'm not sure what's going on there

Reply:
You're right, it does not out of the box [(yet, but it should starting with black 24)](https://github.com/psf/black/issues/1837). I will run it with the mentioned experimental flag and manually check output before commiting.

[flake8-implicit-str-concat](https://pypi.org/project/flake8-implicit-str-concat/) would be useful for such cases, I can add it to the flake8 tox env if you agree.
I'm using pylint on a daily basis, it also [has this rule](https://pylint.readthedocs.io/en/stable/user_guide/messages/warning/implicit-str-concat.html)